www-client/qutebrowser/qutebrowser-1.5.2-r1.ebuild

```diff
@@ -55,6 +55,9 @@ python_install_all() {
 	domenu misc/${PN}.desktop
 	doicon -s scalable icons/${PN}.svg
 
+	exeinto /usr/share/qutebrowser/userscripts/
+	use scripts && doexe misc/userscripts/*
```

Comment:
This is grabbing the `README.md` and any other non-scripts that may be added later.  I'm also mildly concerned that the license for some of these scripts isn't clear.  By nature of them being in the qutebrowser repo can we assume GPL-3?

Reply:
True it's grabbing the `README.md`.  I don't think that's necessarily a problem, the path they are being installed to is not intended to be in  the PATH.  It might make sense to install an explicit list of files to guard against future inclusions though.

As far as licensing goes, I'm not sure.  Some scripts (`qute-pass` for instance) have an explicit Copyright notice, others don't.  I will open an issue on the qutebrowser repo to ask for clarification.  In the meantime we could limit the effect of the `scripts` USE flag to just install those userscripts with an explicit copyright notice.

Reply:
If you limit to the scripts with a known license, I'll happily merge.

Reply:
Installation of userscripts is now limited to those with an explicit license.  I've also added qutebrowser v1.6.0 to the PR.